### DISCUSSION TITLE
SelfControl doesn't work these days

### DISCUSSION THREAD

**Discussion by naw223** [SelfControlApp/selfcontrol#881] [Q&A]
Hello,

I have used selfcontrol for 3 years, no issues. Now for some reason it is not working. Is there any way you can help please? (I use Macbook pro, safari & chrome, MacOS Monterey 12.7.6) Self control doesn't work for both browsers.

Thank you.

**Comment by cstigler** [comment]
Hi, unfortunately I can't provide personal support due to time constraints, but I'd look at what's changed since you used it last! Common culprits are proxies, VPNs, and anything else that affects network connections. Hope you figure it out
